website/pages/docs/platform/k8s/injector/annotations.mdx

```diff
@@ -0,0 +1,143 @@
+---
+layout: docs
+page_title: Agent Sidecar Injector Annotations
+sidebar_title: Annotations
+description: This section documents the configurable annotations for the Vault Agent Injector.
+---
+
+# Annotations
+
+The following are the available annotations for the injector.
+
+## Agent Annotations
+
+- `vault.hashicorp.com/agent-inject` - configures whether injection is explicitly
+  enabled or disabled for a pod. This should be set to a `true` or `false` value.
+  Defaults to `false`.
+
+- `vault.hashicorp.com/agent-inject-status` - blocks further mutations
+  by adding the value `injected` to the pod after a successful mutation.
+
+- `vault.hashicorp.com/agent-configmap` - name of the configuration map where Vault
+  Agent configuration file and templates can be found.
+
+- `vault.hashicorp.com/agent-image` - name of the Vault docker image to use. This
+  value overrides the default image configured in the controller and is usually
+  not needed. Defaults to `vault:1.3.2`.
+
+- `vault.hashicorp.com/agent-init-first` - configures the pod to run the Vault Agent 
```

Comment:
I was slightly confused about his parameter in the k8s PR. You mentioned in the changes that it allows the init container to run first "or last". So if this is false does it always run last?

Reply:
That's correct, I can clarify.

Reply:
Done!